portfolio/src/main/webapp/index.html

```diff
@@ -161,6 +163,47 @@ <h2>~ hiking ~</h2>
         <div id="hiking-map"></div>
       </div>
     </section>
+    <section id="personality">
+      <div>
+        <h2>~ personality type chart ~</h2>
+        <p class="section-descr">
+          What's your Myers-Briggs Type Indicator personality type? There are four dimensions: 
+        </p>
+        <p>
+          <ul>
+            <li>Where you focus your attention - extraversion (E) or introversion(I)</li>
+            <li>The way you take in information - sensing (S) or intuition (N)</li>
+            <li>How you make decisions - thinking (T) or feeling (F)</li>
+            <li>How you deal with the world - judging (J) or perceiving (P)</li>
+          </ul>
+        </p>
+        <p class="section-descr">
+          Take the test <a href="https://www.16personalities.com/free-personality-test">here</a>!
+        </p>
+        <div id="chart-container"></div>
+        <form action="/personality-data" method="POST">
+        <select name="type">
+          <option value="INTJ">Architect (INTJ)</option>
+          <option value="INTP">Logician (INTP)</option>
+          <option value="ENTJ">Commander (ENTJ)</option>
+          <option value="ENTP">Debater (ENTP)</option>
+          <option value="INFJ">Advocate (INFJ)</option>
```

Comment:
I'm an Advocate :D Which one did you say you were?

Reply:
I used to be a Defender, but I recently took it again and now I'm a Virtuoso.